### PR TITLE
community updates issue 442

### PR DESCRIPTION
**Pull request by extrawurst**

### CODE REVIEW

File: draft/2022-05-11-this-week-in-rust.md [hunk 1]
```diff
@@ -27,34 +27,60 @@ and just ask the editors to select the category.
...
 
+* [rust-analyzer changelog #128](https://rust-analyzer.github.io/thisweek/2022/05/09/changelog-128.html)
 * [Fornjot 0.6](https://www.fornjot.app/blog/fornjot-0.6/)
 * [Announcing the Kani Rust Verifier Project](https://model-checking.github.io/kani-verifier-blog/2022/05/04/announcing-the-kani-rust-verifier-project.html)
 * [Slint (UI crate) weekly update -- Version 0.2.2 Release](https://slint-ui.com/thisweek/2022-05-09.html)
+* [Rocket's 2nd v0.5 Release Candidate - Rocket Web Framework](https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/)
+* [rustc_codegen_gcc: Progress Report #11](https://blog.antoyo.xyz/rustc_codegen_gcc-progress-report-11)
+* [GCC Rust Monthly Report #16 April 2022](https://thephilbert.io/2022/05/04/gcc-rust-monthly-report-16-april-2022/)
+* [RepliByte - An open-source tool to seed your dev database with real data](https://www.reddit.com/r/rust/comments/ukmnow/an_opensource_tool_to_seed_your_dev_database_with/)
+* [This Month in Rust OSDev: April 2022](https://rust-osdev.com/this-month/2022-04/)
```
Comment (ericseppanen) [inline draft/2022-05-11-this-week-in-rust.md:50]: Maybe this should go in Newsletters?

File: draft/2022-05-11-this-week-in-rust.md [hunk 2]
```diff
@@ -27,34 +27,60 @@ and just ask the editors to select the category.
...
+* [Introducing Crane: Composable and Cacheable Builds with Cargo and Nix](https://ipetkov.dev/blog/introducing-crane/)
+* [The run-up to v1.0 for Postcard](https://jamesmunns.com/blog/postcard-1-0-run/)
 
 ### Observations/Thoughts
 
 * [Programming languages are platforms, not products](https://kerkour.com/programming-languages-are-platforms)
 * [Introducing Rust in security research](https://tweedegolf.nl/en/blog/71/introducing-rust-in-security-research)
+* [Xilem: an architecture for UI in Rust](https://raphlinus.github.io/rust/gui/2022/05/07/ui-architecture.html)
+* [Modeling Finite State Machines with Rust | Ramnivas Laddad](https://www.ramnivas.com/blog/2022/05/09/fsm-model-rust)
+* [Secure computation in Rust: Using Intel's SGX instructions with Tea...](https://www.notamonadtutorial.com/secure-computation-in-rust-using-intels-sgx-instructions-with-teaclave-and-fortanix/)
```
Comment (ericseppanen) [inline draft/2022-05-11-this-week-in-rust.md:60]: Title was truncated by Discord.

"Secure computation in Rust: Using Intel's SGX instructions with Teaclave and Fortanix"

**Review by ericseppanen — APPROVED** [review]

File: draft/2022-05-11-this-week-in-rust.md (hunk 2 above)
Comment (ericseppanen) [reply draft/2022-05-11-this-week-in-rust.md]: Fixed.

File: draft/2022-05-11-this-week-in-rust.md (hunk 1 above)
Comment (ericseppanen) [reply draft/2022-05-11-this-week-in-rust.md:50]: I'll move this to Newsletters this time, and if you disagree we can discuss on Discord for next time :)

Comment (nellshamrell) [reply draft/2022-05-11-this-week-in-rust.md]: I do think that should be in Newsletters as well

**Pull request merged**